Preventing ball holding and ball manipulation more reliably

medrimonia opened the pull request:
Fixes #230

`game.in_play` should be more robust and easier to understand than the previous
condition.

omichel reviewed (commented):
I am not sure about the changes in resources/webots_ros.
Maybe it is safer to revert them?

medrimonia committed:
Preventing ball holding and ball manipulation more reliably

Fixes #230

`game.in_play` should be more robust and easier to understand than the previous
condition.

medrimonia commented:
You're right that wasn't intentional at all! I've amended the commit.

omichel reviewed (approved):
Thank you.
